### PR TITLE
Add validators exit bus happy path tests

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: tests/regression/test_validator_exit_bus_happy_path.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    processing_deadline_slot: int
+    data_hash: HexString
+    data_submitted: bool
+    data_format: int
+    requests_count: int
+    requests_submitted: int
+
+
+@pytest.fixture()
+def stranger(stranger):
```
Comment (TheDZhon) [inline tests/regression/test_validator_exit_bus_happy_path.py:27]: can use oracle committee member instead (we are not assigning this role for the real setup)

File: tests/regression/test_validator_exit_bus_happy_path.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    contracts.validators_exit_bus_oracle.grantRole(
+        web3.keccak(text="SUBMIT_DATA_ROLE"),
+        stranger,
+        {"from": contracts.agent},
+    )
+    return stranger
+
+
+def _encode_data_from_abi(report_data, abi, func_name):
```
Comment (TheDZhon) [inline tests/regression/test_validator_exit_bus_happy_path.py:36]: maybe extract to the common utils, IDK

**Review by TheDZhon — APPROVED** [review]
:+1: 
Left some insignificant suggestions

File: tests/regression/test_validator_exit_bus_happy_path.py (hunk 1 above)
Comment (skhomuti) [reply tests/regression/test_validator_exit_bus_happy_path.py:27]: yes, it's not used, forgot to remove, thanks!

**Review by bulbozaur — APPROVED** [review]

**Pull request merged**